### DISCUSSION TITLE
Is there a way tp adapt the blueretro project to ps4 or ps3 consoles?

### DISCUSSION THREAD

**Discussion by wolfmanvid01** [darthcloud/BlueRetro#1042] [Ideas]
It's known that those consoles are a pain in the ... when u wanna pair other controllers to them. Ps4 doesnt accept them, and ps3 is dependant on the "playstation" button. Apart that there are custom firmwares for ps3 that doesnt have the bluetooth chip functioning.

Is there anyway to make work the blueretro with these 2 consoles so it can be connected thru usb and then make us able to connect ANY controller we want to them?

**Comment by darthcloud** [comment]
It's something I might do in the future. Not sure when tho.

**Reply by wolfmanvid01** [reply]
it would be great. Ps4 are a pain in the ass to use with other controllers like ps3
